contracts/governance/GovernorBravoDelegate.sol

```diff
@@ -10,10 +10,10 @@ contract GovernorBravoDelegate is GovernorBravoDelegateStorageV1, GovernorBravoE
     string public constant name = "bZx Governor Bravo";
 
     /// @notice The minimum setable proposal threshold
-    uint public constant MIN_PROPOSAL_THRESHOLD = 5150000e18; // 5,150,000 = 0.5% of BZRX
+    uint public constant MIN_PROPOSAL_THRESHOLD = 51500000e18; // 5,150,000 = 0.5% of OOKI
 
     /// @notice The maximum setable proposal threshold
-    uint public constant MAX_PROPOSAL_THRESHOLD = 20600000e18; // 20,600,000 = 2% of BZRX
+    uint public constant MAX_PROPOSAL_THRESHOLD = 206000000e18; // 20,600,000 = 2% of OOKI
```

Comment:
commented values for this and MIN_PROPOSAL_THRESHOLD are not adjusted to 10x of the previous amount.

Reply:
they are not adjusted. but 10x is being taken care of in other place

Reply:
10x is being taken care elswhere

Reply:
ok